content/api/api.adoc

```diff
@@ -65,7 +65,7 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 
 == Other resources
 
-* http://grimoire.arrdem.com/[Grimoire] - community combined cheatsheet + examples
+* https://www.conj.io/[Grimoire] - community combined cheatsheet + examples
```

Comment:
Btw, "community combined cheatsheet + examples" sounds a bit strange to me. If I didn't know what Grimoire was I would probably have struggled to figure this out.

Reply:
feel free to send follow-up pr